tests/nightly/test_large_array.py

```diff
@@ -145,6 +145,11 @@ def test_where():
     res = nd.sparse.where(csr_cond, a, b)
     assert np.sum(res[0].asnumpy() == 1) == b.shape[1]
 
+def test_pick():
```

Comment:
This test is not sufficient. I ran it in current MXNet master and it passed as well.

Reply:
The new test fails on master. Looks good now